### DISCUSSION TITLE
How to update an IP address

### DISCUSSION THREAD

**Discussion by Jonas60526** [fwestenberg/reolink_dev#410] [Q&A]
I moved my camera and the router assigned a new ip address. Can I edit the ip address in the HA files somewhere or do I need to delete and re-add the integration or device?

**Comment by MartinNygaardSchacht** [comment]
You can, but you have to be very careful! I you do this wrong, you could ruin your setup.
Take a complete backup of HA before doing this.

Then:

1. Take a copy of the file:  /config/.storage/core.config_entries
2. Edit the same file, and find the device. It will look like this.
3. Edit the IP Address
4. Save and restart HA. :-)

```
                "entry_id": "fa186b86b1d456acec98fb27d1766da4",
                "version": 1,
                "domain": "reolink_dev",
                "title": "Security Camera Driveway West",
                "data": {
                    "host": "192.168.1.32",
                    "port": 443,
                    "use_https": true,
                    "username": "SUPER_SECRET_PASSWORD",
                    "password": "EVEN_MORE_SECRET_PASSWORD",
                    "channel": 1
```
